microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -18,6 +18,36 @@ when adding an h4 heading, add nav link below the h2
 -->
 
 
+<!-- ====================================================================== -->
+## Show WPF elements on top of the WebView2 layer (WebView2CompositionControl) - INCOMING
```

Comment:
@sivMSFT ( @victorhuangwq @peiche-jessica ) - where to put this .net-only link?
Design options for incoming content in this article include:
* New h2 section
* New h4 section
* Add links in existing tabset
* Add new tabset in existing heading/section
* Add virtual little heading bold pseudo-heading within a tab set

Reply:
@champnic